### PR TITLE
1606 node clients replaces merge

### PR DESCRIPTION
**Pull request by m4sterbunny**
Instances referring to "The Merge" linked to Ethereum docs
The Merge page >> Node clients
Small updates to affected pages

### CODE REVIEW

**Review by bgravenorst — COMMENTED** [review]
Some feedback. I'd suggest a technical review since there's additional content here.

Also, we generally use a 100 character line limit: https://docs-template.consensys.io/contribute/format-markdown#column-limit

File: docs/public-networks/concepts/node-clients.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+### Execution clients
+
+Execution clients, such as Besu, manage the execution layer, including executing transactions and updating the world state. Execution clients serve [JSON-RPC API](../reference/engine-api/index.md) requests and communicate with each other P2P.
+
+Besu is an execution client that you can run with:
+
+- [Any consensus client on Mainnet](../get-started/connect/mainnet.md).
+- [Any consensus client on a testnet](../get-started/connect/testnet.md).
+- [Teku on mainnet](../tutorials/besu-teku-mainnet.md).
+- [Teku on a testnet](../tutorials/besu-teku-testnet.md).
```
[the remark this answers is not in the document]
Comment (m4sterbunny) [reply docs/public-networks/concepts/node-clients.md:45]: @bgravenorst I kept the existing format as it seemed to support the user with relevant links. Not loving the 2 variants of Mainnet within 4 lines!

File: docs/public-networks/concepts/node-clients.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+As as result of [the 2022 Merge](https://ethereum.org/en/upgrades/merge/), Ethereum Mainnet 
+transitioned from Proof of Work (PoW) to [PoS](proof-of-stake/index.md) consensus.
+:::
+
+## Execution and consensus clients
+
+Under PoS, a full Ethereum Mainnet node is a combination of an execution client (previously called 
+an [Eth1 client](https://blog.ethereum.org/2022/01/24/the-great-eth2-renaming/) client) and a 
+consensus client (previously called an 
+[Eth2 client](https://blog.ethereum.org/2022/01/24/the-great-eth2-renaming/)). A
```
Comment (jflo) [inline docs/public-networks/concepts/node-clients.md:26]: Sometimes validators are separate processes from consensus clients, sometimes not. it's an individual deployment detail.

File: docs/public-networks/concepts/node-clients.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+## Execution and consensus clients
+
+Under PoS, a full Ethereum Mainnet node is a combination of an execution client (previously called 
+an [Eth1 client](https://blog.ethereum.org/2022/01/24/the-great-eth2-renaming/) client) and a 
+consensus client (previously called an 
+[Eth2 client](https://blog.ethereum.org/2022/01/24/the-great-eth2-renaming/)). A 
+[validator node](#validator-nodes) also runs the validator client.
+
+Execution and consensus clients communicate with each other using the
```
Comment (jflo) [inline docs/public-networks/concepts/node-clients.md:29]: More strictly, consensus clients drive the execution client via the Engine API. it's kind of a one-way thing.

**Review by jflo — APPROVED** [review]
Minor suggestions, looks fine otherwise.

File: docs/public-networks/concepts/node-clients.md (hunk 2 above)
Comment (m4sterbunny) [reply docs/public-networks/concepts/node-clients.md:26]: I think this is covered in line 66/67, so will resolve here>> Pls reopen if this info needs to be brought up the pg

File: docs/public-networks/concepts/node-clients.md (hunk 3 above)
Comment (m4sterbunny) [reply docs/public-networks/concepts/node-clients.md:29]: great -- update coming through

File: docs/public-networks/concepts/node-clients.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+tags:
+  - public networks
+---
+
+# Node clients
+
+Ethereum's Proof of Stake (PoS) protocol leverages two separate P2P networks supporting 
+separate clients. Execution clients gossip transactions over their network, enabling them to manage 
+their local transaction pool. Consensus clients gossip blocks over their network, enabling consensus 
+and chain growth. A validator node also runs the [validator client]((#validator-clients).
```
Comment (bgravenorst) [inline docs/public-networks/concepts/node-clients.md:14]: Looks like a stray extra bracket in the validator client link.

File: docs/public-networks/concepts/node-clients.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+transitioned from Proof of Work (PoW) to [PoS](proof-of-stake/index.md) consensus.
+:::
+
+## Execution and consensus clients
+
+Under PoS, a full Ethereum Mainnet node is a combination of an execution client (previously called 
+an [Eth1 client](https://blog.ethereum.org/2022/01/24/the-great-eth2-renaming/) client) and a 
+consensus client (previously called an 
+[Eth2 client](https://blog.ethereum.org/2022/01/24/the-great-eth2-renaming/)). The consensus client 
+communicates with the execution client via the [Engine API](../how-to/use-engine-api.md).
```
Comment (bgravenorst) [inline docs/public-networks/concepts/node-clients.md:27]: Was probably in the original text, but we generally try and avoid using "via".

**Review by bgravenorst — APPROVED** [review]
Provided some additional (minor) feedback.

File: docs/public-networks/concepts/node-clients.md (hunk 5 above)
Comment (m4sterbunny) [reply docs/public-networks/concepts/node-clients.md]: image also updated to rmv Engine API bi-directionality as per @jflo

**Comment by m4sterbunny** [comment]
Not merging yet -- want to verify that image update is def fine.
Image now clarifies that the relationship btwn consensus client and execution client is one way via the Engine API

But, that leaves open the Q: how does the consensus client get "the validated data from the execution client"?

**Comment by m4sterbunny** [comment]
OK -- so going to merge -- Execution does emit messages back to Consensus, but apparently the single headed arrow show direction of API requests as standard >> whereas sequence diagram would specify the response

**Pull request merged**